ChiModules/LinearBoltzmannSolver/lua/lbs_groupset_ops.cc

```diff
@@ -737,32 +737,37 @@ int chiLBSGroupsetSetIterativeMethod(lua_State *L)
     exit(EXIT_FAILURE);
   }
 
-  if (iter_method == NPT_CLASSICRICHARDSON)
   {
-    groupset->iterative_method = NPT_CLASSICRICHARDSON;
-  }
-  else if (iter_method == NPT_CLASSICRICHARDSON_CYCLES)
-  {
-    groupset->allow_cycles = true;
-    groupset->iterative_method = NPT_CLASSICRICHARDSON;
-  }
-  else if (iter_method == NPT_GMRES)
-  {
-    groupset->iterative_method = NPT_GMRES;
-  }
-  else if (iter_method == NPT_GMRES_CYCLES)
-  {
-    groupset->allow_cycles = true;
-    groupset->iterative_method = NPT_GMRES;
-  }
-  else
-  {
-    chi_log.Log(LOG_ALLERROR)
-      << "Unsupported iterative method specified in call to "
-      << "chiLBSGroupsetSetIterativeMethod.";
-    exit(EXIT_FAILURE);
+    using LinearBoltzmann::IterativeMethod;
+    if (iter_method == static_cast<int>(IterativeMethod::CLASSICRICHARDSON))
+    {
+      groupset->iterative_method = IterativeMethod::CLASSICRICHARDSON;
+    }
+    else if (iter_method ==
+                static_cast<int>(IterativeMethod::CLASSICRICHARDSON_CYCLES))
+    {
+      groupset->allow_cycles = true;
+      groupset->iterative_method = IterativeMethod::CLASSICRICHARDSON;
+    }
+    else if (iter_method == static_cast<int>(IterativeMethod::GMRES))
+    {
+      groupset->iterative_method = IterativeMethod::GMRES;
+    }
+    else if (iter_method == static_cast<int>(IterativeMethod::GMRES_CYCLES))
+    {
+      groupset->allow_cycles = true;
+      groupset->iterative_method = IterativeMethod::GMRES;
+    }
+    else
+    {
+      chi_log.Log(LOG_ALLERROR)
```

Comment:
Ok. I see. If method is still NONE at this point, then you complain/ Good.

Reply:
Non-the-less. I like workable defaults. Do you think we should make the default GMRES.